AutocompleteClientTests/FW/Logic/Worker/ConstructorIORecommendationsTests.swift

```diff
@@ -257,4 +257,18 @@ class ConstructorIORecommendationsTests: XCTestCase {
         self.constructor.recommendations(forQuery: query, completionHandler: { response in })
         self.wait(for: builder.expectation)
     }
+    
+    func testRecommendations_UsingRecommendationsQueryBuilder_AttachesHiddenFields() {
+        let hiddenFields = ["hidden_field_1", "hidden_field_2"]
+        let query = CIORecommendationsQueryBuilder(podID: "item_page_1")
+            .setHiddenFields(hiddenFields)
+            .build()
+
+        let builder = CIOBuilder(expectation: "Calling Recommendations with hidden fields with the same name should have a multiple facet URL query items", builder: http(200))
+        
+        stub(regex("https://ac.cnstrc.com/recommendations/v1/pods/item_page_1?_dt=\(kRegexTimestamp)&c=\(kRegexVersion)&fmt_options%5Bhidden_fields%5D=hidden_field_1&fmt_options%5Bhidden_fields%5D=hidden_field_2&i=\(kRegexClientID)&key=\(kRegexAutocompleteKey)&num_results=5&s=\(kRegexSession)&section=Products"), builder.create())
```

Comment:
It looks like we're missing `&\(TestConstants.defaultSegments)` at the end here which is causing the test to fail

Reply:
https://github.com/Constructor-io/constructorio-client-swift/pull/223/commits/731e4de30905c7c53803a3b4728e3a26298090ed#diff-a1cb8911026626ff1ea273da2a15eca4e0d0d2050034b918fd4b56f16581f2a0R269